libs/ui/src/remove_card_screen.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Screen } from './screen';
+import { Main } from './main';
+import { Prose } from './prose';
+import { fontSizeTheme } from './themes';
+
+const RemoveCardImage = styled.img`
+  margin: 0 auto -1rem;
+  height: 30vw;
+`;
+
+export function RemoveCardScreen(): JSX.Element {
+  return (
+    <Screen white>
+      <Main centerChild>
+        <Prose textCenter theme={fontSizeTheme.medium}>
+          <p>
+            <RemoveCardImage
+              aria-hidden
+              src="assets/remove-card.svg"
+              alt="Remove Card Icon"
```

Comment:
Since we already have text with clear instructions, this image can probably be considered a decorative one from an accessibility standpoint, in which case we should make the `alt` tag an empty string, `alt=""`. Much more on this here! https://www.w3.org/WAI/tutorials/images/decorative